web/pageObjects/landing_page.ts

```diff
@@ -0,0 +1,9 @@
+import * as defaultPage from "./default_page"
+
+const btnSignInWithEmail = '//main//button[text()="Sign in with email"]'
+const signInForm = '//main/div/form//button[text()="Sign in"]'
```

Comment:
is this element in landing page? Will you use it?

Reply:
aha šitą reikia panaikinti šioj fazėj, nes aš apsirašiau elementą kuris buvo pvz ir tik paskui išsiaiškinau kam jis naudojamas.